<ui version="4.0" >
 <class>IOCTLDialog</class>
 <widget class="QDialog" name="IOCTLDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Device I/O Control</string>
  </property>
  <widget class="QPushButton" name="uiBtnRead" >
   <property name="geometry" >
    <rect>
     <x>310</x>
     <y>50</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Read</string>
   </property>
   <property name="autoDefault" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="uiLEditDeviceID" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>57</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="uiLEditRegister" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>85</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="uiLEditValue" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>113</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="uiBtnSend" >
   <property name="geometry" >
    <rect>
     <x>310</x>
     <y>20</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Send</string>
   </property>
   <property name="autoDefault" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="uiBtnClose" >
   <property name="geometry" >
    <rect>
     <x>310</x>
     <y>110</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
   <property name="default" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="uilabelCamera" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>20</y>
     <width>88</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text" >
    <string>Camera:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="uiComboCamera" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>18</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="uilabelRegNum" >
   <property name="geometry" >
    <rect>
     <x>7</x>
     <y>90</y>
     <width>112</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Register #(hex):</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="uilabelValue" >
   <property name="geometry" >
    <rect>
     <x>7</x>
     <y>118</y>
     <width>112</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text" >
    <string>Value(hex):</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="uilabelDeviceID" >
   <property name="geometry" >
    <rect>
     <x>7</x>
     <y>60</y>
     <width>112</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text" >
    <string>Device ID(hex):</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>uiComboCamera</tabstop>
  <tabstop>uiLEditDeviceID</tabstop>
  <tabstop>uiLEditRegister</tabstop>
  <tabstop>uiLEditValue</tabstop>
  <tabstop>uiBtnSend</tabstop>
  <tabstop>uiBtnRead</tabstop>
  <tabstop>uiBtnClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>uiBtnClose</sender>
   <signal>clicked()</signal>
   <receiver>IOCTLDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>351</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel" >
     <x>203</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
